admin/lib/azure-upload.js

```diff
@@ -1,42 +1,31 @@
 'use strict'
 
 const path = require('path')
-const azure = require('azure-storage')
 const moment = require('moment')
-const blobService = azure.createBlobService()
-const winston = require('winston')
+const blobService = require('../services/data-access/azure-blob.data.service')
 
 // Files get uploaded to this container.  dns naming conventions.
 const container = 'check-development-app-upload-files'
```

Comment:
can you rename this to something more appropriate while you are working 'in this area' please.   always bugged me.  
maybe `admin-app-uploaded-files` ?

Reply:
@GuyHarwood done